packages/cloud-cognitive/src/components/Tearsheet/_tearsheet.scss

```diff
@@ -238,6 +240,8 @@
     }
 
     // and restore the 'light' prop in case light fields are wanted
+    .#{$prefix}--pagination--light,
+    .#{$prefix}--pagination__control-buttons--light,
```

Comment:
I think these two classes are not in fact used, because, rather inconsistently, Pagination does not have a 'light' prop.

```suggestion
```